src/NuGet.Clients/PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -265,6 +266,159 @@ internal Lazy<Task<BackgroundLoaderResult>> BackgroundLoader
 
         public Uri IconUrl { get; set; }
 
+        // same URIs can reuse the bitmapImage that we've already used.
+        static Dictionary<Uri, BitmapImage> bitmapImageCache = new Dictionary<Uri, BitmapImage>();
```

Comment:
Question: Why do we vacate the cache when moving out of the browse tab? Should we keep it there until we close the whole UI?